classes/console/console.gd

```diff
@@ -28,14 +28,13 @@ func run_command(cmd: String):
 					Singleton.log_msg("No second argument.", Singleton.LogType.ERROR)
 					return
 				match args[1]:
-					"t1":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_1.tscn"
-					"t1r2":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_2.tscn"
-					"t1r3":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_3.tscn"
-					"t1r4":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_4.tscn"
+					"tutorial_1":
```

Comment:
Perhaps have the option of using the shorter name `"t1"`?

You can do this with the line `"t1", "tutorial_1":`. This will allow both as options.